.github/workflows/code-quality.yml

```diff
@@ -9,8 +9,22 @@ jobs:
     - uses: actions/setup-python@v2
       with:
         python-version: "3.12"
-    - run: pip install --upgrade pip
-    - run: pip install "ruff<1" "black<23" "isort<6" 
-    - run: ruff check .
-    - run: black --check .
-    - run: isort --check --profile black .
+
+    - name: Install linters
+      run: |
+        pip install --upgrade pip
+        pip install "ruff" "black" "isort" "mypy"
+        pip install SQLAlchemy
```

Comment:
Is any reason to add these libs to ci workflow?

Reply:
mypy needs it to check